src/open_inwoner/ssd/xml.py

```diff
@@ -45,11 +47,13 @@ def format_name(first_name: str, voorvoegsel: str, last_name: str):
     first_name_initials = [name[0] + "." for name in first_names]
     first_name_formatted = " ".join(first_name_initials)
```

Comment:
Old feedback: for single-use iterables it's nicer to use generators, ex: a generator expression instead of a list comprehension. There is no need to allocate memory for all the new items at once if we're only looking at one at a time.

```py
    # not square brackets but parenthesis
    first_name_initials = (name[0] + "." for name in first_names)
    first_name_formatted = " ".join(first_name_initials)

    # the parenthesis aren't actually part of the expression itself but for syntax reasons so sometimes can be left out like when calling a function, although in practice Black will probably do a linebreak
    first_name_formatted = " ".join(name[0] + "." for name in first_names)
```